DEPS

```diff
@@ -176,6 +176,11 @@ hooks = [
     "condition": 'checkout_mac and host_os != "mac" and checkout_dmg_tool',
     'action': ['build/mac/cross-compile/build-libdmg-hfsplus.py', 'third_party/libdmg-hfsplus']
   },
+  {
+    'name': 'update_group_policy',
+    'pattern': '.',
+    'action': ['python3', 'build/util/update_group_policy.py']
```

Comment:
this is not great. A developer expects that modifying a source/policy file will trigger a rebuild. This won't happen in the current approach. Consider doing the copying during the build process, not during the sync.

you can do this by overriding `policy_templates.py` `main()` function.

Reply:
I concur with this too.

Reply:
OK good call - done. Need to try it out 😄